addons/resource.language.en_gb/resources/strings.po

```diff
@@ -466,17 +474,17 @@ msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#123"
-msgid "Copy these files?"
+msgid "Would you like to copy the selected file(s)?"
 msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#124"
-msgid "Move these files?"
+msgid "Would you like to move the selected file(s)?"
 msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#125"
-msgid "Delete these files? - Deleting files cannot be undone!"
+msgid "Would you like to delete the selected file(s)?[CR][CR]Deleting any file(s) can't be undone!"
```

Comment:
I don't like having hardcoded line breaks. Also we could probably write:
"Would you like to delete the selected file(s)? Warning - this action cannot be undone!"
@jjd-uk ?